### DISCUSSION TITLE
Change font family

### DISCUSSION THREAD

**Discussion by m2mathew** [reduxjs/redux-devtools#1013] [Q&A] [answered]
Great changes to the Redux dev tools! I can use the Themes and Color Scheme (I see you, `unikitty` 🦄) to modify spacing and colors. However the font choice is a bit hard to read for me, even when I scale it up past 100%.

Are we able to change the font family to something different?

**Comment by markerikson** [comment]
See https://github.com/reduxjs/redux-devtools/issues/995 for some existing related discussion.

**Reply by m2mathew** [reply]
Thanks, @markerikson. I like the way you think. 🤓

**Comment by Methuselah96** [comment, marked as answer]
Yes, I hope to add the ability to change the font type and size shortly or at least provide a better default. Track https://github.com/reduxjs/redux-devtools/issues/995 to know when that goes live.
